CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# Contribution Guidelines
+
+Thanks for your interest in contributing to dwave-examples! Before getting
+started, we encourage you to take a look at this document for some guidelines.
+
+## What We're Looking For
+
+While we have a preference for application oriented examples, we welcome all
+examples that demonstrate use of the D-Wave system.
+
+For a list of existing examples, take a look
+[here](https://cloud.dwavesys.com/leap/examples/).
+
+## Steps for Contributing
+
+1. Download this repository for a template and implement your example on a
+   personal repository. Note: To simplify the code reviewing process, we
+   encourage you to use a development branch. This way, the code may be reviewed
+   as a pull request.
+
+2. Open an issue on this repository and link to your example. See the following
+   section for what to include in your issue.
+
+3. Once your example has been reviewed, ____?
+
+## Issue Template
+
+**Description**
+
+A clear and concise description of the concepts in your project and the problem 
+your code solves. Please include any references you used.
+
+**How to run the example**
+
+A short description of how to run your code, including an example.
```

Comment:
This might be redundant if they include this information as part of the README.

Reply:
Agree

Reply:
Agreed and removed. Also moved this whole section to a bullet point in the section above.